test/python/compiler/test_transpiler.py

```diff
@@ -90,6 +90,7 @@
 from qiskit.transpiler.passmanager_config import PassManagerConfig
 from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager, level_0_pass_manager
 from qiskit.transpiler.target import (
+    InstructionDurations,
```

Comment:
Lint is likely to yell at you here 🙂.

Reply:
Oh, right, this was from when I was trying to cover `InstructionDurations` in the old test before I realized that it was not really directly relevant for `transpile` since `InstructionProperties` is in seconds and `transpile` does not take `instruction_durations` as an input.